app/models/concerns/image_file.rb

```diff
@@ -64,7 +64,18 @@ def reset_image_attributes
 
   def detect_image_attributes
     return if !original_url || (width && height && format)
-    info = FastImage.new(original_url)
+    info = nil
+    begin
+      fastimage_options = {
+        timeout: 10,
+        raise_on_failure: true,
+        http_header: { 'User-Agent' => 'Mozilla/5.0 (Macintosh; Intel Mac OS X) PRX Feeder/1.0' }
+      }
+      info = FastImage.new(original_url, fastimage_options)
+    rescue FastImageException => err
```

Comment:
Do you need to define this anywhere? And do we want a test for it? (Just tried testing it out and got *** NameError Exception: uninitialized constant ImageFile::FastImageException ***)